### PR TITLE
Improve manage_all.py code

### PR DESCRIPTION
**Pull request by DTadrzak**
Signed-off-by: DTadrzak <daniel.tadrzak@intel.com>

### CODE REVIEW

**Comment by DTadrzak** [comment]
@PiotrProkop

**Comment by pawelsocha** [comment]
manage_all raise exception No such file or directory without any information when kubectl is no available /usr/bin/.

Maybe make more descriptive exception?

**Comment by DTadrzak** [comment]
it's not a manage_all issue. It should be handled by stackanetes code. Mange_all only execute stackanetes.

**Comment by mateusz-blaszkowski** [comment]
Actually if you're improving manage_all.py, it'd be nice to explain this:

`services = [file_name[:-4] for file_name in services]`

(os.path.splitext would do the job)

File: manage_all.py [hunk 1]
```diff
@@ -33,20 +33,18 @@ def main():
...
 
 
 def manage_services(services, manage_type):
+    cmd = ['stackanetes', '--debug', manage_type, 'service_type']
     for service_name in services:
-        cmd = ['stackanetes', '--config-dir',
-               '/etc/stackanetes', '--debug', manage_type]
-        cmd.append(service_name)
-        print cmd
+        cmd[-1] = service_name
```
Comment (pawelsocha) [inline manage_all.py]: change it to cmd + [ service_name ] 
will make cmd variable const between loop iterations

**Pull request merged**